static/script/devices/mediaplayer/samsung_streaming.js

```diff
@@ -415,6 +418,11 @@ require.def(
                 if (this.getState() !== MediaPlayer.STATE.BUFFERING) {
                     return;
                 }
+                
+                if (!this._isInitialBufferingFinished() && this._nextSeekingTo !== null) {
+                    this._deferSeekingTo = this._nextSeekingTo;
```

Comment:
I feel like it would simplify things if the `_isInitialBufferingFinished` check was done in `_deferredSeek`, and we always used `_deferSeekingTo` instead of `_nextSeekingTo`.